qemu/tests/cfg/spice.cfg

```diff
@@ -7,10 +7,10 @@
     display_vm2 = vnc
     vga_vm2 = cirrus
     full_screen = no
-    virtio_ports_virt-tests-vm1 = "vdagent"
-    virtio_port_type_virt-tests-vm1 = "serialport"
-    virtio_port_chardev_virt-tests-vm1 = "spicevmc"
-    virtio_port_name_prefix_virt-tests-vm1 = "com.redhat.spice."
+    serials_virt-tests-vm1 += " vdagent"
```

Comment:
`serials += " vdagent"`

Reply:
@yanan-fu I think that should have `_virt-tests-vm1` as previous defined: `virtio_ports_virt-tests-vm1`, that's for a specific vm, better not change original configuration.

Reply:
@vivianQizhu  oh~, yes, you are right, i didn't notice there are not only one vm.